File: VB/TextEditSample/MainForm.vb
Imports System.Drawing

Namespace TextEditSample
    Partial Public Class MainForm
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
            EditorHelpers.CreatePersonPrefixImageComboBox(prefixImageComboboxEdit.Properties, Nothing)
            Me.employeeBindingSource.Add(EmployeeDataHelper.CreateDefaultEmployee())
        End Sub
        Protected Overrides Sub OnLookAndFeelChangedCore()
            MyBase.OnLookAndFeelChangedCore()
            ClientSize = dataLayoutControl1.Root.MinSize
        End Sub
    End Class
End Namespace

File: CS/TextEditSample/MainForm.cs
using System.Drawing;

namespace TextEditSample {
    public partial class MainForm : DevExpress.XtraEditors.XtraForm {
        public MainForm() {
            InitializeComponent();
            EditorHelpers.CreatePersonPrefixImageComboBox(prefixImageComboboxEdit.Properties, null);
            this.employeeBindingSource.Add(EmployeeDataHelper.CreateDefaultEmployee());
        }
        protected override void OnLookAndFeelChangedCore() {
            base.OnLookAndFeelChangedCore();
            ClientSize = dataLayoutControl1.Root.MinSize;
        }
    }
}